cfme/tests/configure/test_server_name.py

```diff
@@ -45,7 +46,8 @@ def _ensure_name_reset():
     )
 
     # CFME updates about box only after any navigation BZ(1408681) - closed wontfix
-    navigate_to(appliance.server, 'Dashboard')
+    view = navigate_to(appliance.server, 'Dashboard')
+    view.browser.refresh()
```

Comment:
Why do we need this refresh?

Reply:
BZ 1408681, navigation alone is not consistent enough.